wpeverest/everest-forms
# Fix - Empty checkboxes value in view entry form details

### All Submissions:






### Changes proposed in this Pull Request:

Previously, When the user fills up the form, checkbox values were sent to the email but empty values shown in **_view entry form details_**. This PR fixes the issue to show the value inserted by the user in **_view entry form details_** as well.

Ref: https://wordpress.org/support/topic/empty-6/


### How to test the changes in this Pull Request:

1. Open the related form (ex. **Contact Form**) and fill up the data and submit.
2. Go to **Entries** submenu under the **Everest Forms** menu.
3. Click on **View** link under **Actions** column in the table.
4. That's all. Now you can see the changes which now shows the values of checkbox fields as well.

### Types of changes:




### Other information:




### Changelog entry

> Empty checkboxes value in view entry form details.
